### PR TITLE
Add more detail on agent

### PR DESCRIPTION
**Pull request by jamiehannaford**
The current README references a non-existent file in `examples`, so I've pointed to the right file. I've also added clarification on how the agent runs and other minor things 😃

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,29 +14,37 @@ on the node before rebooting.
...
+- The `locksmithd.service` systemd unit on each machine should be masked and stopped in systemd
+
+To unmask a service, run `systemctl unmask <name>`.
+To enable a service, run `systemctl enable <name>`.
+To start/stop a service, run `systemctl start <name>` or `systemctl stop <name>` respectively.
 
 ## Usage
 
-To start `update-operator` and `update-agent`:
+To start the `update-operator` DaemonSet, run:
```
Comment (euank) [inline README.md]: This one's a deployment

File: README.md [hunk 2]
```diff
@@ -13,29 +14,37 @@ on the node before rebooting.
...
 
 ```
-# Open examples/components.yaml and edit the image tag.
-kubectl create -f examples/components.yaml
+kubectl create -f examples/update-operator.yaml
```
Comment (euank) [inline README.md]: Oh, also, should the comment still be there?
Right now it still has "${TAG}", though I'd be in favour of updating TAG to be the latest release instead as an alternate solution.

Comment (jamiehannaford) [inline README.md]: Yeah, I like the idea of using `latest` by default since I assume that's what 99% of users will want.

**Comment by jamiehannaford** [comment]
Thanks for reviewing @euank. I've made the suggested changes.

**Pull request merged**